rgw/v2/tests/s3_swift/configs/test_cloud_transition_headobject_false.yaml

```diff
@@ -13,3 +13,4 @@ config:
     create_object: true
```

Comment:
Please update the typo of script file its suppose to be : # script: test_cloud_transition.py
changes needed in all 3 config file where config "enable_encryption: false" added

Reply:
Done